lib/vrl/compiler/Cargo.toml

```diff
@@ -8,7 +8,7 @@ publish = false
 [dependencies]
 diagnostic = { package = "vrl-diagnostic", path = "../diagnostic" }
 parser = { package = "vrl-parser", path = "../parser" }
-shared = { path = "../../shared", default-features = false }
+shared = { path = "../../shared", default-features = false, features = ["conversion"] }
```

Comment:
Was this related?

Reply:
Yessir. Turns out this feature flag is required if you `cargo check` in the compiler crate directly. I'll make a sweep with cargo hack to flush out more of these kinds of build bugs.

Reply:
Just noting that this should no longer happen now that https://github.com/timberio/vector/pull/8658 is merged.